src/main/kotlin/com/exactpro/th2/ws/client/Main.kt

```diff
@@ -84,8 +85,9 @@ fun main(args: Array<String>) = try {
     val settings = factory.getCustomConfiguration(Settings::class.java, mapper)
     val eventRouter = factory.eventBatchRouter
     val messageRouter = factory.messageRouterMessageGroupBatch
+    val grpcRouter = factory.grpcRouter
 
-    run(settings, eventRouter, messageRouter, handler) { resource, destructor ->
+    run(settings, eventRouter, messageRouter, grpcRouter, handler) { resource, destructor ->
         resources += resource to destructor
     }
```

Comment:
```suggestion
    run(settings, eventRouter, messageRouter, grpcRouter, handler) { resource, destructor ->
        resources += resource to destructor
    }
    exitProcess(0)
```
exitProcess is required to run shutdown hook and close all resources